scripts/pylint_extensions_test.py

```diff
@@ -259,3 +259,65 @@ def test(test_var_one, test_var_two, self): #@
             ),
         ):
             checker_test_object.checker.visit_functiondef(functiondef_node2)
+
+
+class SingleCharAndNewlineAtEOFCheckerTests(unittest.TestCase):
+
+    def test_checks_single_char_and_newline_eof(self):
+        checker_test_object = testutils.CheckerTestCase()
+        checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.SingleCharAndNewlineAtEOFChecker)
+        checker_test_object.setup_method()
+        node1 = astroid.scoped_nodes.Module(name='test', doc='Custom test')
```

Comment:
Try giving names which explain the variable. For instance you can use: `node_with_missing_newline_at_eof`.

Reply:
Thanks for pointing that out @YashJipkate! Can you please try renaming them too?

Reply:
Ah okay I'll rename them too!
Thanks.